Nos/Views/NewPostView.swift

```diff
@@ -11,16 +11,14 @@ import SwiftUINavigation
 
 struct NewPostView: View {
     private var keyPair: KeyPair? {
-        get {
-            if let privateKeyData = KeyChain.load(key: KeyChain.keychainPrivateKey),
-               let hexString = NSString(data: privateKeyData, encoding: NSUTF8StringEncoding) as? String,
-               let pair = KeyPair(privateKeyHex: hexString) {
-                return pair
-            } else {
-                print("Error loading key pair from keychain")
-                return nil
-            }
+		if let privateKeyData = KeyChain.load(key: KeyChain.keychainPrivateKey) {
```

Comment:
Could you please change your Xcode preferences to indent using 4 spaces instead of tabs?